ui/calendar.js

```diff
@@ -183,14 +190,14 @@ return $.widget( "ui.calendar", {
 	_buildMultiplePicker: function() {
 		var headerClass,
 			html = "",
-			currentDate = this.date,
-			months = this.date.months( this.options.numberOfMonths - 1 ),
+			currentDate = this.viewDate,
+			months = this.viewDate.months( this.options.numberOfMonths - 1 ),
 			i = 0;
 
 		for ( ; i < months.length; i++ ) {
 
 			// TODO: Shouldn't we pass date as a parameter to build* fns instead of setting this.date?
```

Comment:
Not sure if this comment was intended as a "fix me" or a "improve me" type of a comment. Could be removed if we are fine with the current implementation from a code quality perspective.